File: src/core/logger.js
export function createLogger({ verbose = false } = {}) {
  return {
    info(message) {
      console.log(message);
    },
    warn(message) {
      console.warn(message);
    },
    debug(message) {
      if (verbose) {
        console.error(`[debug] ${message}`);
      }
    }
  };
}

File: go_issue_agent/core/logger.py
def create_logger(verbose: bool = False):
    class Logger:
        def info(self, message: str) -> None:
            print(message)

        def warn(self, message: str) -> None:
            print(message)

        def debug(self, message: str) -> None:
            if verbose:
                print(f"[debug] {message}")

    return Logger()
